docs/source/getting-started.md

```diff
@@ -0,0 +1,194 @@
+# Getting Started with ExecuTorch
+This section is intended to describe the necessary steps to take PyTorch model anda run it using ExecuTorch. To use the framework, you will need to take the following steps:
+- Install the ExecuTorch python package and runtime libraries.
+- Export the PyTorch model for the target hardware configuration.
+- Run the model using the ExecuTorch runtime APIs.
+
+<hr/>
+
+## Installation
+To use ExecuTorch, you will need to install both the Python package and the appropriate platform-specific runtime libraries.
+
+Pip is the recommended way to install the ExecuTorch python package. This package includes the dependencies needed to export a PyTorch model, as well as Python runtime bindings for model testing and evaluation.
+```
+pip install executorch
+```
+
+#### System Requirements
+- Python 3.10 - 3.12
+- g++ version 7 or higher, clang++ version 5 or higher, or another C++17-compatible toolchain.
+- Linux or MacOS operating system (ARM or x86).
+  - Windows is supported via WSL.
+
+#### Runtime Libraries
+ExecuTorch provides a native C++ runtime API, as well as platform and language-specific bindings for Python, Android, and iOS.
+
+##### Python
```

Comment:
Would swap the Python and C++ sections for ordering, since Python is the least important runtime out of these

Reply:
I ended up moving it to the exporting section. In my experience the python bindings are heavily used when exporting a model, testing, and doing server evals, so I think it's worth calling out here. Though it's arguably much closer to the AOT flow, so I've moved it up and renamed this section to "Running the Model on Device".